ingeniamotion/pdo.py

```diff
@@ -301,6 +299,34 @@ def high_precision_sleep(duration: float) -> None:
             while duration - (time.perf_counter() - start_time) > 0:
                 pass
 
+        def __set_watchdog_timeout(self) -> None:
+            if self._watchdog_timeout is None:
+                self._watchdog_timeout = max(
+                    self.DEFAULT_WATCHDOG_TIMEOUT,
+                    self._refresh_rate * self.PDO_WATCHDOG_INCREMENT_FACTOR,
+                )
+                is_watchdog_timeout_manually_set = False
+            else:
+                is_watchdog_timeout_manually_set = True
+            try:
+                for servo in self._net.servos:
+                    servo.set_pdo_watchdog_time(self._watchdog_timeout)
+            except AttributeError as e:
+                max_pdo_watchdog = re.findall("wd_time_ms is limited to (.+) ms", e.__str__())
```

Comment:
Once the pysoem PR is accepted this needs to go

Reply:
Yes, the issue is already created (INGM-582).